edge-agent/docker/linux/amd64/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG base_tag=3.1.19-alpine3.14
+ARG base_tag=3.1-alpine
```

Comment:
The AMD64 RocksDB build should match the OS in the containers it's being installed into. This ensures the library dependencies don't cause loading issues at runtime. I'm probably being overly cautious, but pointing it to 3.14 for this release is the safest way to ensure no trouble. 
RockDB Dockerfile is here: https://github.com/Azure/iotedge/blob/release/1.2/edge-util/docker/linux/amd64/Dockerfile

Reply:
> Do you think it is a reasonable thing to do to have the two container be using the same image?

I probably wasn't clear before, yes, I think this may require additional changes to the RocksDB Dockerfile, but that is a completely valid way to handle this.

Reply:
@darobs The Dockerfile you referenced above is already based on Alpine 3.14. Are there other changes you think we'll need to make?

Reply:
Per brief discussion yesterday during the standup, I think we should create an checkin-E2E-gate test to verify that RocksDb can correctly be loaded upon the module's deployment. This will future-proved us with 1) RocksDb load 2) RocksDb base OS docker build.

Reply:
We could base the RocksDb build container off of `mcr.microsoft.com/dotnet/aspnet:3.1-alpine`. That way it's exactly the same *and* will be more likely to be found when we update the dotnet base.